Base/EventNotification/StateTransition.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO.BACnet.EventNotification.EventValues;
+
+namespace System.IO.BACnet.EventNotification
+{
+    public class StateTransition : EventNotificationData
+    {
+        private readonly EventNotificationData _eventData;
+
+        #region EventNotificationData properties
```

Comment:
Not a fan of regions tbh, especially if there is nothing to separete like in this case. And if there is something to separete, I prefer to decouple it and not have to use regions.

Reply:
I use regions just to collapse code - in this case code that is rather uninteresting.

Reply:
Yeah I get it, I just don't like the the code inside the region and even when the region is collaped I still know it's there 🤣